### ISSUE TITLE
Signing RPM files using Mariner certs

### ISSUE THREAD

**Issue by epananth** [dotnet/arcade#7672]
- [ ] This issue is blocking 
- [ ] This issue is causing unreasonable pain

Related issue (dotnet/core-eng/issues/13072)

We need to sign RPM files using Mariner certs.

Related doc=>
cc: @adiaaida @jonfortescue

**Comment by epananth** [comment]
I was under the impression that we need to use Signtool and a different cert to sign to sign RPMs files. But after looking at the doc I am not sure that is the case. I found 4 cases in the doc

1)  Mariner Team Builds on Behalf-Of  - in this owning team does not want to create/ publish the packages.
2)  Owning Team Publishes Content on Mariner’s Behalf - owning team will created, validate and publish the packages.
3) 1st Party Team Produces Mariner Binaries, Mariner Packages and Publishes Publicly -published to packages.microsoft.com.
4) 1st Party Team Produces Mariner binaries End-to-End, Releases Privately - owning team will create , validate packages using mariner tool kit. Also Signing is not needed in this situation since the package is essentially side-loaded to the device. 

I did send an email to Mariner team if we can just sign the files with the mariner cert using arcade signtool. I have not got any response yet. 

This is pretty confusing so want to know your thoughts.
